src/cli/commands/force/lightning/local/device/__tests__/list.test.ts

```diff
@@ -52,23 +52,23 @@ const androidDevices: AndroidVirtualDevice[] = [
         'pixel-xl-path',
         'Google APIs',
         'Android 9',
-        Version.from('28')
+        Version.from('28')!
     ),
     new AndroidVirtualDevice(
         'Nexus_5X',
         'Nexus 5X',
         'nexus-5x-path',
         'Google APIs',
         'Android 10',
-        Version.from('29')
+        Version.from('29')!
     ),
     new AndroidVirtualDevice(
         'Pixel_4_XL',
         'Pixel 4 XL',
         'pixel-4-xl-path',
         'Google APIs',
         'Android 11',
-        Version.from('30')
+        Version.from('30')!
```

Comment:
What does a `!` do? My guess is that the version "must be" and no lower or no higher?

Reply:
`Version.from()` now returns an optional (`Version | null`), where it will be `null` if the version is un-parseable. The `!` just says that I know this version is parseable, so force-cast it to `Version`.